distributions

```diff
@@ -3,6 +3,6 @@ Label: git-lfs
 Suite: stable
 Codename: debian
 Version: 1.0
-Architectures: i386 amd64 source
+Architectures: i386 amd64 arm arm64 source
```

Comment:
Same comment about `arm` here, but otherwise this looks fine.

Reply:
I updated it to the correct matching architectures that are specified for ARM.  I used the architectures from [here](https://www.debian.org/releases/stretch/). Also add an little execlude for jessie because it doesn't support `arm64`.